crypto/x509/by_store.c

```diff
@@ -112,6 +112,7 @@ static int by_store_ctrl_ex(X509_LOOKUP *ctx, int cmd, const char *argp,
     switch (cmd) {
     case X509_L_ADD_STORE:
         if (argp != NULL) {
+            CHECK_AND_SKIP_CASE_PREFIX(argp, "file:");
```

Comment:
It's not 100% clear to me if this is a suitable fix or not. Perhaps @levitte can comment.

I wonder about pulling this commit out into a separate PR - because I could approve everything else.

Reply:
Thanks for your swift review and nice to hear that you could already approve everything else.

The existing `s_client` doc says:
```
With URIs in the C<file:> scheme, this acts as B<-chainCAfile> or
B<-chainCApath>, depending on if the URI indicates a directory or a
single file.
See L<ossl_store-file(7)> for more information on the C<file:> scheme.
```
which is not true or at least misleading because in fact the `file:` scheme is not (yet) recognised for these options.
So we should either remove the mention of `file:` (which I'd then do also for  my proposed doc extensions),
or we fix this as part of this PR in some way.

Of course I'm open to any better way of fixing the issue.
The way I propose doing it would also fix this for the respective API-level functions, not only for the CLI.

@levitte can you please share your view?

Reply:
Ping @levitte